bin/admin/unlock-home.sh

```diff
@@ -22,6 +36,20 @@ fi
 
 if [ -e "$MOUNTPOINT/allowkeeper" ] && mountpoint -q /home ; then
     echo "Already unlocked and mounted"
+    # Stop the banner seal service to ensure banner is in unsealed state
+    if command -v systemctl >/dev/null 2>&1; then
+        if systemctl stop osh-seal-banner.service; then
+            echo "SSH banner switched to unsealed state"
+        else
+            echo "Warning: Could not stop osh-seal-banner service"
+        fi
+    else
+        if service osh-seal-banner stop; then
+            echo "SSH banner switched to unsealed state"
+        else
+            echo "Warning: Could not stop osh-seal-banner service"
+        fi
+    fi
```

Comment:
It might be a good idea to take this portion of code and move it to a dedicated function such as `update_banner()`? This way we would avoid code duplication between this portion of code and the same one lines 12-25, what do you think?

It's very readable and straightforward, but I'm afraid if it later needs to be patched, we could forget to update one of the two sections!